test/functional/shared/resources/kubemetadata_httproute_test.go

```diff
@@ -40,11 +40,11 @@ func Test_KubeMetadataHTTPRoute(t *testing.T) {
 
 	expectedLabels := map[string]string{
 		"app.kubernetes.io/managed-by": "radius-rp",
-		"app.kubernetes.io/name":       "ctnr-rte-kme",
+		"app.kubernetes.io/name":       "ctnr-rte-kme-ctnr",
 		"app.kubernetes.io/part-of":    "corerp-app-rte-kme",
-		"radapp.io/application":        "corerp-app-rte-kme",
-		"radapp.io/resource":           "ctnr-rte-kme",
-		"radapp.io/resource-type":      "applications.core-httproutes",
+		"radius.dev/application":       "corerp-app-rte-kme",
+		"radius.dev/resource":          "ctnr-rte-kme-ctnr",
+		"radius.dev/resource-type":     "applications.core-containers",
```

Comment:
How did these work before?

Reply:
before we had the httproutes. Therefore, the labels such as "radapp.io/resource-type": "applications.core-httproutes" were used in code the retrieve/ process logic related to them. But now, we are moving away from httproutes and the service creation happens as part of rendering a container with ports construct.  Therefore, these labels had to be updated in code and the testcases too. Updates to code was done in as earlier PR by Faraz.

Reply:
https://github.com/radius-project/radius/pull/6241#discussion_r1383682750 
The rror is the bicep file uses "global" as application ID which is invalid.